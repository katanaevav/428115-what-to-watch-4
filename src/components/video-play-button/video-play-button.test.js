import React from "react";
import renderer from "react-test-renderer";
import VideoPlayButton from "./video-play-button.jsx";

it(`Render video play button`, () => {
  const tree = renderer.create(
      <VideoPlayButton
        onPlayButtonClick = {() => {}}
        isPlaying = {false}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});

it(`Render video pause button`, () => {
  const tree = renderer.create(
      <VideoPlayButton
        onPlayButtonClick = {() => {}}
        isPlaying = {true}
      />
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
