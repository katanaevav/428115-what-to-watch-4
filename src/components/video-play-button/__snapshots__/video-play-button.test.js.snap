// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render video pause button 1`] = `
<button
  className="player__play"
  onClick={[Function]}
  type="button"
>
  <svg
    height="21"
    viewBox="0 0 14 21"
    width="14"
  >
    <use
      xlinkHref="/sprite.svg#pause"
    />
  </svg>
  <span>
    Pause
  </span>
</button>
`;

exports[`Render video play button 1`] = `
<button
  className="player__play"
  onClick={[Function]}
  type="button"
>
  <svg
    height="19"
    viewBox="0 0 19 19"
    width="19"
  >
    <use
      xlinkHref="/sprite.svg#play-s"
    />
  </svg>
  <span>
    Play
  </span>
</button>
`;
